lib/log.js

```diff
@@ -70,12 +72,14 @@ Lug.prototype.begin = function (op, request) {
   this.logger.debug(op)
 }
 
-Lug.prototype.event = function (name, data) {
+Lug.prototype.event = function (name, request, data) {
   var e = {
     event: name,
     data: unbuffer(data)
   }
-  process.stdout.write(JSON.stringify(e) + '\n')
+  data.metricsContext = metricsContext.add({},
```

Comment:
Actually it looks like `unbuffer` in this file refers to a local function that does modify the object in place by default.  It's still clearer to spell out the full name though, so I'll change it.